### ISSUE TITLE
line 65: 25580 Segmentation fault

### ISSUE THREAD

**Issue by chenQ1114** [google-research/xtreme#54]
Hi 

I am trying to build the baseline training model for the task udpos. When I run the bash scripts/train.sh [model] udpos command, I always got such error, which seems indicate the problem from last line of parameters for $REPO/third_party/run_tag.py. 

Changing the action='store_true' to default=True in parser.add_argument or adjust the  train_batch_size cannot solve it.
How to fix it? thanks!

Here is the output file:

Fine-tuning bert-base-multilingual-cased on udpos using GPU 0
Load data from /home/mwp141/Working/xtreme/download, and save models to /home/mwp141/Working/xtreme/outputs-temp
12/04/2020 11:27:19 - INFO - root -   Input args: Namespace(adam_epsilon=1e-08, cache_dir=None, config_name='', data_dir='/home/mwp141/Working/xtreme/download/udpos/udpos_processed_maxlen128', device=device(type='cuda'), do_eval=True, do_lower_case=False, do_predict=True, do_predict_dev=True, do_train=True, eval_all_checkpoints=True, eval_patience=-1, evaluate_during_training=True, few_shot=-1, fp16=False, fp16_opt_level='O1', gradient_accumulation_steps=4, init_checkpoint=None, labels='/home/mwp141/Working/xtreme/download/udpos/udpos_processed_maxlen128/labels.txt', learning_rate=2e-05, local_rank=-1, log_file='/home/mwp141/Working/xtreme/outputs-temp/udpos/bert-base-multilingual-cased-LR2e-5-epoch2-MaxLen128/train.log', logging_steps=50, max_grad_norm=1.0, max_seq_length=128, max_steps=-1, model_name_or_path='bert-base-multilingual-cased', model_type='bert', n_gpu=1, no_cuda=False, num_train_epochs=2.0, output_dir='/home/mwp141/Working/xtreme/outputs-temp/udpos/bert-base-multilingual-cased-LR2e-5-epoch2-MaxLen128', overwrite_cache=False, overwrite_output_dir=True, per_gpu_eval_batch_size=8, per_gpu_train_batch_size=5, predict_langs='af,ar,bg,de,el,en,es,et,eu,fa,fi,fr,he,hi,hu,id,it,ja,kk,ko,mr,nl,pt,ru,ta,te,th,tl,tr,ur,vi,yo,zh', save_only_best_checkpoint=True, save_steps=500, seed=1, server_ip='', server_port='', tokenizer_name='', train_langs='en', warmup_steps=0, weight_decay=0.0)
12/04/2020 11:27:19 - WARNING - __main__ -   Process rank: -1, device: cuda, n_gpu: 1, distributed training: False, 16-bits training: False
/home/mwp141/Working/xtreme/scripts/train_udpos.sh: line 65: 25580 Segmentation fault      (core dumped) python3 $REPO/third_party/run_tag.py 
--data_dir $DATA_DIR 
--model_type $MODEL_TYPE 
--labels $DATA_DIR/labels.txt 
--model_name_or_path $MODEL 
--output_dir $OUTPUT_DIR 
--max_seq_length $MAX_LENGTH 
--num_train_epochs $NUM_EPOCHS 
--gradient_accumulation_steps $GRAD_ACC 
--per_gpu_train_batch_size $BATCH_SIZE 
--save_steps 500 
--seed 1 
--predict_langs $LANGS 
--log_file $OUTPUT_DIR/train.log 
--learning_rate $LR
